proposals/semi-auto-properties.md

```diff
@@ -12,61 +12,31 @@ In these cases by now you always have to create an instance field and write the
-```
-
-...
+[§14.7.1](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1471-general) *Properties - General*
+
+> A *property_initializer* may only be given for ~~an automatically implemented property, and~~  **a property that has a backing field that will be emitted and the property either does not have a setter, or its setter is auto-implemented. The *property_initializer*** causes the initialization of the underlying field of such properties with the value given by the *expression*.
+
+[§14.7.4](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1474-automatically-implemented-properties) *Automatically implemented properties*
+
+> An automatically implemented property (or ***auto-property*** for short), is a non-abstract non-extern
+> property with ~~semicolon-only accessor bodies. Auto-properties must have a get accessor and can optionally have a set accessor.~~
+> **either or both of:**
+> 1. **an accessor with a semicolon-only body**
+> 2. **usage of the `field` contextual keyword within the accessors or**
+>    **expression body of the property. The `field` identifier is only considered the `field` keyword when there is**
+>    **no existing symbol named `field` in scope at that location.**
+> 
+> Auto-properties shall have a get accessor and may optionally have a set accessor. 
+>
+> When a property is specified as an automatically implemented property, a hidden **unnamed** backing field is automatically
+> available for the property ~~, and the accessors are implemented to read from and write to that backing field~~.
+> **For auto-properties, any semicolon-only `get` accessor is implemented to read from, and any semicolon-only `set` accessor to write to its backing field.**
+> If the auto-property has ~~no set accessor~~ **only a semicolon-only get accessor**, the backing field is considered `readonly` ([§14.5.3](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/classes.md#1453-readonly-fields)).
+> Just like a `readonly` field, a getter-only auto-property **(without a set accessor or an init accessor)** can also be assigned to in the body of a constructor 
+> of the enclosing class. Such an assignment assigns directly to the ~~readonly~~ backing field of the property.
+> 
+> **The backing field can be referenced directly using the `field` keyword within all accessors and within the property expression body.**
+> **Because the field is unnamed, it cannot be used in a `nameof` expression.**
 
 ````diff
 +The following example:
```

Comment:
If you don't mind, I think we should also remove the surrounding diff block from here, and the + at the start of every line, and just indicate this as being a wholly new section that should be added to the standard.

Reply:
Removed `diff`.